### DISCUSSION TITLE
Connecting PlanetScale to BigQuery?

### DISCUSSION THREAD

**Discussion by drakedeatonuk** [planetscale/discussion#474] [Q&A] [answered]
Currently investigating creating a GCP datastream from a PlanetScale DB into BigQuery.

I've seen zero guidance or support on if this is possible anywhere online. I feel like it should be surely.

Does anyone have any advice they can offer?

Thanks in advance!

**Comment by jgreet** [comment, marked as answer]
It looks like Datastream requires binlog access for MySQL sources. PlanetScale can't provide binlog access currently, so I don't think using their Datastream connector is going to work.

See https://cloud.google.com/datastream/docs/configure-your-source-mysql-database#selfhostedmysql

You should be able to PlanetScale Connect and one of the supported ETL tools to sync data to BigQuery. I believe we've used Airybte for this exact purpose.

https://planetscale.com/docs/integrations/airbyte

**Comment by drakedeatonuk** [comment]
gotchya - thanks @jgreet will have a look into this! 🙏
